src/openrct2/ride/TrackData.cpp

```diff
@@ -5427,6 +5427,765 @@ constexpr static uint8_t TrackTypeToSpinFunction[TrackElemType::Count] = {
     NO_SPIN, NO_SPIN, NO_SPIN, NO_SPIN, NO_SPIN, NO_SPIN, NO_SPIN, NO_SPIN, NO_SPIN
 };
 
+template<const int32_t TConstant> static int32_t EvaluatorConst(const int16_t)
```

Comment:
```suggestion
template<int32_t TConstant> static int32_t EvaluatorConst(const int16_t)
```
Template arguments are not mutable.